### PR TITLE
Create SimpleRegressorList Class #14

### PR DESCRIPTION
**Pull request by sh-biswas**
Created the SimpleRegressorList Class and added tests as well.

I haven't gotten to run the test file on my own machine, as I wanted to use Github actions to see the output, but they were modeled off of the SimpleRegressor tests. 

Here's an example of the regression list based off of the boston dataset:
Screen Shot 2021-01-23 at 12 04 06 AM
Screen Shot 2021-01-23 at 12 04 18 AM
Screen Shot 2021-01-23 at 12 04 57 AM

### CODE REVIEW

**Comment by sh-biswas** [comment]
I am running into an error that says "AttributeError: 'SimpleRegressorList' object has no attribute 'logger'", but I'm not sure why this is happening, as that try-except code has not changed from the SimpleClassifierList code.

Edit: I've added the logger attribute for SimpleRegressorList. I suppose it just wasn't needed in SimpleClassifierList, as there was no logger attribute present in that class, but the logging was present.

**Comment by sh-biswas** [comment]
There is another issue, however. Whenever I find the p-values of the first 2 regressors in the list, it seems the p-value for the best regressor seems to bounce between 2 different values, but the p-value for the 2nd best regressor remains consistent. My assumption for the p-values was that the best regressor would have the highest p-value (please correct me if I'm wrong). The regressor ranks remain the same. I believe this might affect the testing.
Here's an example:

First fit:
Screen Shot 2021-01-23 at 8 22 43 AM

Second fit:
Screen Shot 2021-01-23 at 8 23 55 AM

File: tests/regressors/simple_regressor_list_tests.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        rgr0 = rgr_list.pop()
+
+        pred0_y = rgr0.predict(true_x)
+        pred1_y = rgr1.predict(true_x)
+
+        stat0, p0 = levene(true_y, pred0_y)
+        stat1, p1 = levene(true_y, pred1_y)
+
+        self.assertTrue(p0 > 0.05)
+        self.assertTrue(p1 < p0)
```
Comment (skekre98) [inline tests/regressors/simple_regressor_list_tests.py]: Let's remove this from the test when comparing in terms of ranking. Levene is just a form of validation so it is indeed possible for p1 > p0. We can perhaps have Levene as the metric for ranking in the future.

File: tests/regressors/simple_regressor_list_tests.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
```
Comment (skekre98) [inline tests/regressors/simple_regressor_list_tests.py]: same as above

**Comment by skekre98** [comment]
Hi @sh-biswas, this is great work! I think my review comments should answer your question. Please let me know if it is still unclear.

**Comment by sh-biswas** [comment]
Yes, thank you for the clarification! I have removed those assert statements.

**Review by skekre98 — APPROVED** [review]
Cool, looks good to me! Once the tests pass we should be good to merge 👍

**Pull request merged**